sql/V1_10__credit_classes.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE IF NOT EXISTS
+  credit_classes (
+    TYPE TEXT NOT NULL,
+    block_height BIGINT NOT NULL,
+    tx_idx SMALLINT NOT NULL,
+    msg_idx SMALLINT NOT NULL,
+    chain_num SMALLINT NOT NULL,
+    TIMESTAMP timestamptz,
+    tx_hash TEXT NOT NULL,
+    credit_class_id TEXT NOT NULL,
+    issuer_address TEXT NOT NULL,
```

Comment:
I think we should be consistent with the on-chain field names, i.e. `class_id` and `issuer` here.

```suggestion
    class_id TEXT NOT NULL,
    issuer TEXT NOT NULL,
```